engine/src/flutter/lib/ui/window/platform_configuration.h

```diff
@@ -49,6 +49,9 @@ enum class AccessibilityFeatureFlag : int32_t {
   kHighContrast = 1 << 5,
   kOnOffSwitchLabels = 1 << 6,
   kNoAnnounce = 1 << 7,
+  kNoAutoPlayAnimatedImages = 1 << 8,
+  kNoAutoPlayVideoPreviews = 1 << 9,
+  kNonBlinkingCursor = 1 << 10,
```

Comment:
we call this deterministicCursor in framework, should use this word instead

https://api.flutter.dev/flutter/widgets/EditableText/debugDeterministicCursor.html